crates/bevy_ecs/src/component.rs

```diff
@@ -1,9 +1,10 @@
 //! Types for declaring and storing [`Component`]s.
 
 use crate::{
+    self as bevy_ecs,
```

Comment:
What this bein used for?

Reply:
Our derive macros don't know the difference between `bevy_ecs` and a third party crate, so we need to include this anytime we use a derive from within `bevy_ecs`. It's a very annoying issue that, as far as I know, can't be fixed without changes to rustc.

Reply:
Yeah I've ruin into this before and never found a satisfying workaround.

Reply:
Ah ya, forgot about that req, looks good to me :)